ui/components/MeshModelRegistry/MesheryTreeView.js

```diff
@@ -176,70 +196,87 @@ const MesheryTreeViewItem = ({
             }
             check={true}
             onClick={() => {
+              getModelByNameTrigger({
+                name: versionedModelDef.name,
+                params: {
+                  version: versionedModelDef.model.version,
+                  components: true,
+                  relationships: true,
+                },
```

Comment:
By default lazy loading does not use caching, please add this prop to make sure it is using it's caching behaviour
```suggestion
                }, true
```